.github/actions/upstream-test/action.yml

```diff
@@ -43,11 +43,24 @@ runs:
       - name: Installing Dependencies in Docker image
         uses: firehed/multistage-docker-build-action@v1
         with:
-          repository: ghcr.io/${{ github.repository_owner }}/dagmc-ci-ubuntu-${{ inputs.ubuntu_version }}-${{ inputs.compiler}}-geant4_${{ inputs.geant_version }}-hdf5_${{ inputs.hdf5_version }}-moab_${{ inputs.moab_version }}
+          repository: >
```

Comment:
Let's wait for all the new Docker images to finish building, then we can rerun the Linux tests to see what happens.

Reply:
I looked at the failed logs and saw that the failure is because this multi-line container name is resolving with spaces in the name, so it definitely needs to be fixed.